org.wireshark.qt.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.wireshark.qt
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: wireshark
+rename-icon: wireshark
+rename-desktop-file: wireshark.desktop
+rename-appdata-file: wireshark.appdata.xml
+finish-args:
+  - --socket=x11
+  # Because the file chooser somehow doesn't show the files on the host, we export some directory for reading pcap files
+  - --filesystem=xdg-public-share
+  # - --share=network
+
+
+modules:
+  - name: libpcap
+    cleanup:
+        - "/include"
+        - "/share"
+    sources:
+        - type: archive
+          url: https://www.tcpdump.org/release/libpcap-1.9.0.tar.gz
+          sha256: 2edb88808e5913fdaa8e9c1fcaf272e19b2485338742b5074b9fe44d68f37019
+
+
+  - name: wireshark
+    config-opts:
+        - --with-qt
```

Comment:
This should not be needed. It's the default.

Reply:
Sorry, my mistake. This is an autotools option, and that build system doesn't exist anymore.